apps/dh-app-e2e/src/plugins/index.js

```diff
@@ -27,12 +27,9 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-const { preprocessTypescript } = require('@nrwl/cypress/plugins/preprocessor');
```

Comment:
info: this function is deprecated because it's unnecessary in recent versions of Nx Cypress